### PR TITLE
ci: jira-issue-creator

### PR DESCRIPTION
**Pull request by Mercy811**
Create a Jira issue when a new issue is created

### CODE REVIEW

File: .github/workflows/jira-issue-create-template.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Template for jira issue creation in order to creates jira tickets for new github issues to help triage
+# Notice that the change of this temple might effect jira issue creation for other repositories
+name: Jira Issue Creator
```
Comment (justin-fiedler) [inline .github/workflows/jira-issue-create-template.yml:3]: Is there a reason to redefine this action in this repro? For most other repos we import the template from Amplitude-TypeScript

E.g.
https://github.com/amplitude/Amplitude-Kotlin/blob/main/.github/workflows/jira-issue-create.yml

File: .github/workflows/jira-issue-creator.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  issues:
+    types: [opened]
+  workflow_call:
+    inputs:
+      label:
+        type: string
+
+jobs:
+  call-workflow-passing-data:
+    uses: amplitude/amplitude-browser-sdk-gtm-template/.github/workflows/jira-issue-create-template.yml@main
```
Comment (justin-fiedler) [inline .github/workflows/jira-issue-creator.yml:14]: Reuse existing template?
```suggestion
    uses: amplitude/Amplitude-TypeScript/.github/workflows/jira-issue-create-template.yml@main
```

**Review by justin-fiedler — CHANGES_REQUESTED** [review]

File: .github/workflows/jira-issue-create-template.yml (hunk 1 above)
Comment (Mercy811) [reply .github/workflows/jira-issue-create-template.yml:3]: Thanks for catching this.

**Review by justin-fiedler — APPROVED** [review]
Thanks @Mercy811 !

**Pull request merged**